.github/workflows/linux.yml

```diff
@@ -26,11 +26,23 @@ jobs:
           python-version: '3.9'
       - name: Install dependencies
         run: |
-          python -m pip install --upgrade meson PyYAML
           sudo apt update
           sudo apt install -y \
-            doxygen libxcb-xkb-dev valgrind ninja-build \
-            libwayland-dev wayland-protocols bison graphviz
+            doxygen libxcb-xkb-dev valgrind ninja-build meson \
+            libwayland-dev wayland-protocols bison graphviz curl
+          python -m pip install --upgrade PyYAML
```

Comment:
this rework means we're no longer testing against the latest released meson but against whatever our ubuntu version ships. That's generally not ideal - any specific reason why you reshuffled this? I don't see the old failing logs so it's unclear

Reply:
Fixed.

Testing locally with podman, it uses `root`, which cannot use pip to update system dependencies. How I reproduce the exact same config as the CI locally?

Reply:
hmm, good question, I don't actually know how to pull the same containers (VMs?) github uses. if you just need an extra user you can create that inside podman and then `su` to it.